packages/grafana-schema/src/raw/composable/logs/panelcfg/x/LogsPanelCfg_types.gen.ts

```diff
@@ -11,6 +11,8 @@
 
 import * as common from '@grafana/schema';
 
+export const pluginVersion = "10.1.0-pre";
```

Comment:
do we need to have this file checked into source control? Will be a lot of files to update every release :)

Reply:
Yes, that its true.

In that case I can create a `plugin_config.cue` file with the common information for all the plugins. It could be useful  if we want to add more information in the future. At the moment we can set only the version and we only update one file for every Grafana version. In that case, where is the best place to generate this file?

And because some plugins have their own version set in `plugin.json` file, does it have sense if I set the pluginVersion only to the ones that have it set?

Reply:
I think we could generate these files only when publishing grafana-schema. I mean - otherwise they are not relevant in core (see @sdboyer comments in my attempt of using those in core https://github.com/grafana/grafana/pull/69382). 

But I think having the `pluginVersion` const in the gen file is going to be very relevant for https://github.com/grafana/scenes/pull/225

Reply:
So... only we need that in @grafana/schema?

Reply:
Plugin version? Most probably yes

Reply:
i'm gonna guess that generating this version string might be a more generalized pattern of interest when it comes to codegen for plugins, so CCing @marefr and @mckn for visibility